tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -191,13 +191,16 @@ private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leade
         }).collect(Collectors.toList());
     }
 
-    private static long durationMs(long timestampMs) {
-        Instant instant = Instant.ofEpochMilli(timestampMs);
+    /* test */ static long delayMs(long timestampMs, String desc) {
```

Comment:
nit: In Kafka, we usually put the test comment on one line above, i.e.
```
// visible for testing
static long delayMs(long timestampMs, String desc) {
```

Reply:
Also, the method `delayMs` doesn't look right to me. I did a google search, it looks like the `relative time` is the correct name for it. Check [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/format) and [here](https://momentjs.com/docs/#/displaying/fromnow/). So, maybe `relativeTimeMs`?

Reply:
Sure, I like relativeTimeMs. Thanks.